rollup.config.js

```diff
@@ -0,0 +1,23 @@
+const resolve = require('@rollup/plugin-node-resolve');
+const babel =  require("@rollup/plugin-babel");
+const commonjs = require('@rollup/plugin-commonjs');
+
+// rollup.config.js
+/**
+ * @type {import('rollup').RollupOptions}
+ */
+const config = {
+  input: 'lib/index.js',
+  output: {
+    name: "esptooljs",
+    file: 'bundle.js',
+    format: 'es',
+  },
+  plugins: [
+    resolve(),
+    commonjs(),
+    babel({ exclude: 'node_modules/**', babelHelpers: "runtime", skipPreflightCheck: true }),
```

Comment:
What is Babel used for? TypeScript can already convert to a target JavaScript version

Reply:
Babel is used to modify ES6 JS code into [browser friendly code](https://babeljs.io). Without this, JS code was not working properly in the browser.

Reply:
Ok 👍

In that case we can have TypeScript output modern JS as suggested in https://github.com/espressif/esptool-js/pull/53#discussion_r1028122793